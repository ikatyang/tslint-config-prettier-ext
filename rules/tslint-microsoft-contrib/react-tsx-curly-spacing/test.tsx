const a = <Hello name={firstname} />;
const b = <Hello name={ firstname} />;
const c = <Hello name={firstname } />;
