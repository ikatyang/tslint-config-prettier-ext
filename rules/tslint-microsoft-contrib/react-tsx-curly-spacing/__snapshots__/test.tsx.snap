// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (react-tsx-curly-spacing) /rules/tslint-microsoft-contrib/react-tsx-curly-spacing/test.tsx[2, 23]: There should be no space after '{'
ERROR: (react-tsx-curly-spacing) /rules/tslint-microsoft-contrib/react-tsx-curly-spacing/test.tsx[3, 23]: There should be no space before '}'

======
no error

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
const a = <Hello name={firstname} />;
const b = <Hello name={ firstname} />;
const c = <Hello name={firstname } />;

======
const a = <Hello name={firstname} />;
const b = <Hello name={firstname} />;
const c = <Hello name={firstname} />;

>>>>>> after
"
`;
